### PR TITLE
Added Project Calico networking plugin tutorial

### PR DESCRIPTION
**Pull request by ozdanborne**
Planning on having this tutorial ready to merge on Wednesday of next week (7/16). We are currently working on cleaning up and passing conformance tests now, but I wanted to get it up early so that I can get any feedback in time to definitely have it merged next week. Thanks!

### CODE REVIEW

**Comment by WIZARD-CXY** [comment]
I think 15.04 ubuntu is a little higher. Our lab uses ubuntu 14.04 which uses upstart. Anyway looking forward to your work.

**Comment by vmarmol** [comment]
Needs a rebase btw :)

**Comment by caseydavenport** [comment]
Does anyone know the state of the conformance tests on master?  We've seen intermittent failures even on Master using the Vagrant/Salt demo.

Should we expect the tests to pass on a working cluster?

**Comment by erictune** [comment]
conformance test should be run from conformance-test-v1 branch.
Currently passes on GCE.  Working to get it to pass on vagrant, but not there yet.  Don't know about other platforms.

**Comment by caseydavenport** [comment]
@erictune Thanks for the info, glad to know its not just us.

With regards to new getting started guides, is it a requirement that they pass the conformance tests?  It does not appear that any of the existing guides are marked as passing.

Would we, for example, be able to clean up this doc such that it is ready to merge, and add conformance in a later PR?

**Comment by erictune** [comment]
It is not a requirement at this time.  We may later remove guides that
don't pass, with some suitable warning.

**Comment by erictune** [comment]
however, you definitely need to sort out the CLA.

File: docs/getting-started-guides/ubuntu-calico.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# Untar it
+tar -xf kubernetes.tar.gz
+tar -xf kubernetes/server/kubernetes-server-linux-amd64.tar.gz
+kubernetes/cluster/ubuntu/build.sh
+# Add binaries to /usr/bin
+sudo cp -f binaries/master/* /usr/bin
+sudo cp -f binaries/kubectl /usr/bin
+```
+>You can customize your etcd version,  k8s version by changing variable `ETCD_VERSION` and `K8S_VERSION` in build.sh, default etcd version is 2.0.9, and K8s version is 0.18.0.
+
```
Comment (erictune) [inline docs/getting-started-guides/ubuntu-calico.md]: you say 0.18 here and 0.20 above.

File: docs/getting-started-guides/ubuntu-calico.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+6.) Use calicoctl to add an IP Pool. We must specify where the etcd daemon is in order for calicoctl to communicate with it.
+```
+ETCD_AUTHORITY=<MASTER_IP>:4001 calicoctl pool add 172.17.0.0/16
+```
+
+#### Launch other Services With Kubernetes
+At this point, you have a fully functioning cluster running on kubernetes with a master and 2 nodes networked with Calico. Lets start some services and see that things work.
+
+`$ kubectl get nodes`
```
Comment (erictune) [inline docs/getting-started-guides/ubuntu-calico.md]: Is this incomplete?  You talk about starting services but then don't.

Comment (djosborne) [inline docs/getting-started-guides/ubuntu-calico.md]: Yes, very incomplete. My initial comment should have emphasized that we are still working on tuning this guide in conjunction with the conformance

File: docs/getting-started-guides/ubuntu-calico.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
+Kubernetes Deployment On Bare-metal Ubuntu Nodes with Calico Networking
+------------------------------------------------
+
+## Introduction
+
+This document describes how to deploy kubernetes on ubuntu bare metal nodes with Calico Networking plugin. See [projectcalico.org](http://projectcalico.org) for more information on what Calico is, and [the calicoctl github](https://github.com/Metaswitch/calico-docker) for more information on the command-line tool, `calicoctl`.
```
Comment (Symmetric) [inline docs/getting-started-guides/ubuntu-calico.md]: s/kubernetes/Kubernetes/ throughout.

**Comment by djosborne** [comment]
@erictune I've updated this pull request and am ready for a review

**Comment by djosborne** [comment]
@erictune / @google-admin / admins -  Do you know when you will be able to look at this? Its ready to go, I'd love to have it in soon to make the 1.0 release.

**Comment by lxpollitt** [comment]
@erictune Can you accept this PR please?  We are exceptionally keen to have this in before 1.0 launch party tomorrow.

**Comment by djosborne** [comment]
@vmarmol I've rebased. Can we have this merged today?

**Comment by lxpollitt** [comment]
Note that this is a docs only change.  The failing shippable and travis stuff looks to be related to the base branch, not to this change.

**Comment by thockin** [comment]
You need to run hack/run-gendocs.sh, which has some markdown sanitization logic.

**Comment by djosborne** [comment]
@thockin Ran gendocs and applied the resulting changes.

**Comment by thockin** [comment]
LGTM

**Comment by thockin** [comment]
@krousey this can go in when you have green (though shippable is not telling us anything for this)

**Comment by krousey** [comment]
This shippable failure should be fixed when #11629 gets merged.

**Pull request merged**